servers/navigation_server_2d.h

```diff
@@ -318,6 +335,21 @@ class NavigationServer2D : public Object {
 	NavigationServer2D();
 	~NavigationServer2D() override;
 
+	enum ProcessInfo {
+		INFO_ACTIVE_MAPS,
+		INFO_REGION_COUNT,
+		INFO_AGENT_COUNT,
+		INFO_LINK_COUNT,
+		INFO_POLYGON_COUNT,
+		INFO_EDGE_COUNT,
+		INFO_EDGE_MERGE_COUNT,
+		INFO_EDGE_CONNECTION_COUNT,
+		INFO_EDGE_FREE_COUNT,
+		INFO_OBSTACLE_COUNT,
+	};
```

Comment:
We need to consider if we really want to stick with the same naming and enums. There have been multiple PRs that added new properties and debug info that at least I did withdraw because the timing for compatibility breaking was a problem. Before we now repeat all that old again for 2D and cement it further we might want to reconsider that.

Reply:
Agreed, open for any suggestions, would suggest keeping the old data in `Performance` as js